.github/workflows/codeql-analysis.yml

```diff
@@ -15,7 +15,7 @@ on:
     branches:
       - develop
   push:
-    # Dependabot triggered push events have read-only access, but uploading code
+    # Dependabot-triggered push events have read-only access, but uploading code
```

Comment:
Won't this filter down when https://github.com/cisagov/skeleton-generic/pull/202 is merged and a lineage PR is generated in this project?

Reply:
Yes, but I thought you would want to keep them in sync for now.  I will close this pull request.